docs/examples/bond-vlan.yaml

```diff
@@ -15,7 +15,7 @@ spec:
         mode: balance-rr
         options:
           miimon: '140'
-        slaves:
+        ports:
```

Comment:
`git grep -n slaves` shows a few more references in our documentation, might be worth changing those at the same time 

```
docs/user-guide/102-configuration.md:169:        slaves:
docs/user-guide/102-configuration.md:214:        slaves:
docs/user-guide/103-troubleshooting.md:75:      autoconnect_slaves: <enum NM_SETTING_CONNECTION_AUTOCONNECT_SLAVES_YES of type NM.SettingConnectionAutoconnectSlaves>
docs/user-guide/bond0-eth1-eth2_up.yaml:17:        slaves:
```

Reply:
Done